src/main.ts

```diff
@@ -111,25 +120,43 @@ async function postDiffComment(diffs: Diff[]): Promise<void> {
   const shortCommitSha = String(sha).substr(0, 7);
 
   const diffOutput = diffs.map(
-    ({ app, diff }) => `    
+    ({ app, diff, error }) => `   
+
 Diff for App: [\`${app.metadata.name}\`](https://${ARGOCD_SERVER_URL}/applications/${
       app.metadata.name
-    }) App sync status: ${app.status.sync.status === 'Synced' ? 'Synced ✅' : 'Out of Sync ⚠️'}
+    }) ${error ? ' Error 🛑' : ''}
+App sync status: ${app.status.sync.status === 'Synced' ? 'Synced ✅' : 'Out of Sync ⚠️'}
```

Comment:
Will there be a sync status if there's an error too?

Reply:
Yes, because the app is still present in ArgoCD and has a sync status there.